pkg/proxy/winkernel/proxier.go

```diff
@@ -173,15 +196,97 @@ func conjureMac(macPrefix string, ip net.IP) string {
 	return "02-11-22-33-44-55"
 }
 
-func newEndpointInfo(ip string, port uint16, isLocal bool, hns HostNetworkService) *endpointsInfo {
+func (proxier *Proxier) endpointsMapChange(oldEndpointsMap, newEndpointsMap proxy.EndpointsMap) {
+	for svcPortName := range oldEndpointsMap {
+		proxier.onEndpointsMapChange(&svcPortName)
+	}
+
+	for svcPortName := range oldEndpointsMap {
```

Comment:
Looks like this is a duplicate of the lines above
```suggestion
	for svcPortName := range newEndpointsMap {
```

Reply:
good catch. It was a bug